core-container/entry.sh

```diff
@@ -303,15 +331,42 @@ fi
 if [ "$Namespace" == "" ]; then
```

Comment:
What sets `$Namespace` ?

Reply:
The Namespace refers to the space framework namespace where the provider objects reside.  It is set in the values.yaml and passed to the entry.sh as an environment variable in the deployment script.  The code above is there defensively in case it is not set.  Regarding the second part (why use a different namespace than the one the chart is deployed in) 
I am leaving this to @MikeSpreitzer and @ezrasilvera to argue. I personally agree they should be one and the same.

Reply:
What makes `entry.sh` get a setting for `$Namespace`? I see that `values.yaml` has a top-level entry `spaceNamespace`. I do not see the path to the former from the latter. I see https://github.com/effi-ofer/edge-mc/blob/PR2/core-helm-chart/templates/deployment.yaml#L105 but that is setting a _different_ environment variable.

Reply:
As discussed I removed the spaceNamespace and reverted to using the helm install namespace (Release.Namespace).